### ISSUE TITLE
Source Control - Error creating New Repo

### ISSUE THREAD

**Issue by ThunderFrame** [rubberduck-vba/Rubberduck#2353]
I've configured my Settings, and have an open project.

I tried to Init a new Repo, and got this error text:

"An unknown error occurred"

Logs set to Debug:

```text
2016-11-09 09:17:59.6932;ERROR;Rubberduck.Common.WinAPI.RawInput;Error sizing the rawinput buffer: The handle is invalid;System.ComponentModel.Win32Exception (0x80004005): The handle is invalid
2016-11-09 09:17:59.7163;ERROR;Rubberduck.Common.WinAPI.RawInput;Error sizing the rawinput buffer: The handle is invalid;System.ComponentModel.Win32Exception (0x80004005): The handle is invalid
2016-11-09 09:18:06.7303;DEBUG;Rubberduck.UI.Command.MenuItems.ParentMenus.ParentMenuItemBase;(55864401) Executing click handler for menu item '&Source Control', hash code 30067389;
2016-11-09 09:18:38.7537;ERROR;Rubberduck.UI.SourceControl.SourceControlViewViewModel;Ninject.ActivationException: Error activating ISourceControlProvider
No matching bindings are available, and the type is not self-bindable.
Activation path:
  1) Request for ISourceControlProvider

Suggestions:
  1) Ensure that you have defined a binding for ISourceControlProvider.
  2) If the binding was defined in a module, ensure that the module has been loaded into the kernel.
  3) Ensure you have not accidentally created more than one kernel.
  4) If you are using constructor arguments, ensure that the parameter name matches the constructors parameter name.
  5) If you are using automatic module loading, ensure the search path and filters are correct.

   at Ninject.KernelBase.Resolve(IRequest request)
   at Ninject.ResolutionExtensions.GetResolutionIterator(IResolutionRoot root, Type service, Func`2 constraint, IEnumerable`1 parameters, Boolean isOptional, Boolean isUnique)
   at Ninject.Extensions.Factory.Factory.InstanceResolver.Get(Type type, String name, Func`2 constraint, IParameter[] parameters, Boolean fallback)
   at Ninject.Extensions.Factory.StandardInstanceProvider.GetInstance(IInstanceResolver instanceResolver, MethodInfo methodInfo, Object[] arguments)
   at Ninject.Extensions.Factory.FactoryInterceptor.Intercept(IInvocation invocation)
   at Castle.DynamicProxy.AbstractInvocation.Proceed()
   at Castle.Proxies.ISourceControlProviderFactoryProxy.CreateProvider(IVBProject project)
   at Rubberduck.UI.SourceControl.SourceControlViewViewModel.InitRepo() in c:\Users\Andrew\Documents\GitHub\Rubberduck\RetailCoder.VBE\UI\SourceControl\SourceControlViewViewModel.cs:line 641
   at Rubberduck.UI.SourceControl.SourceControlViewViewModel.<.ctor>b__0(Object _) in c:\Users\Andrew\Documents\GitHub\Rubberduck\RetailCoder.VBE\UI\SourceControl\SourceControlViewViewModel.cs:line 68
   at Rubberduck.UI.Command.DelegateCommand.ExecuteImpl(Object parameter) in c:\Users\Andrew\Documents\GitHub\Rubberduck\RetailCoder.VBE\UI\Command\DelegateCommand.cs:line 26
   at Rubberduck.UI.Command.CommandBase.Execute(Object parameter) in c:\Users\Andrew\Documents\GitHub\Rubberduck\RetailCoder.VBE\UI\Command\CommandBase.cs:line 54;
```

**Comment by Hosch250** [comment]
So, you hard-coded the type to be injected?  Now this is less ready to support other SC systems, like SVN.  Whatever is going on, this isn't the solution.

**Comment by mansellan** [comment]
This should be fixed as of #3401
